ZTP.md

```diff
@@ -60,8 +60,24 @@ Secure Zero Touch Provisioning (SZTP) adds a bootstrap server to DHCP-based ZTP
 - Both IPv4 and IPv6 DHCP discovery and ZTP provisioning should be supported.
 - For environments that are predominately IPv6 based, we'll need a solution that supports SLAAC.
 - ZTP process and results, both success and failures, must be logged.
+- DHCP is untrusted, only used to point to Bootstrap server.
+- Device must trust network and Network must trust device.
```

Comment:
Either capitalize both instances of `network` or leave them both without capitals.

Reply:
on it